fala/apps/adviser/tests/test_search_page_journeys_playwright.py

```diff
@@ -22,9 +14,25 @@ def test_postcode_search_journey(self):
         ]
         for postcode in test_cases:
             with self.subTest(postcode=postcode):
-                self.visit_search_page()
-                self.page.get_by_label("Postcode").fill(postcode)
-                self.page.get_by_role("button", name="Search").click()
-                expect(self.page.locator("h1")).to_have_text("Search results")
-                expect(self.page.get_by_role("listitem").filter(has_text=f"Postcode: {postcode}")).to_be_visible()
-                expect(self.page.get_by_text("in order of closeness")).to_be_visible()
+                page = self.visit_results_page(postcode)
+                expect(page.h1).to_have_text("Search results")
+                expect(page.listitem_for(postcode)).to_be_visible()
+                expect(page.item_from_text("in order of closeness")).to_be_visible()
+
+    def test_invalid_postcode_journey(self):
+        page = self.invalid_postcode_search()
```

Comment:
i would bring the lines from this invalid method directly into the test so it is readable. We aren't reusing this one and it is directly what we want to test.

Reply:
Agree, will move this across 🙂